src/components/navigation/Navigation.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  FaTwitter, FaFacebook, FaInstagram, FaGoogle,
+} from 'react-icons/fa';
+import { RiPinterestFill } from 'react-icons/ri';
+import carLogo from './car_logo.webp';
+import './navigation.css';
+
+const Navigation = () => (
```

Comment:
- [ ] I couldn't find a way to log out of the app.

Kindly add a way to logout of the app so users can be switched, and multiple users can have their own reservations.

If a way to logout exists, kindly make it more obvious so your users can take advantage of it.

Tip:  It is still possible to logout of the app manually by deleting the localStorage in the browser's devtools:

![image](https://user-images.githubusercontent.com/57421823/223581905-84a4a65e-3b29-49ce-a6ff-a016cb42cd69.png)